app/views/rsvps/_header.html.erb

```diff
@@ -4,4 +4,9 @@
   </div>
   <h5><%= @event.presented_by %></h5>
   <h1><%= @event.name %></h1>
+  <% unless @event.description.blank? %>
```

Comment:
#minor maybe better to have the positive check: `if @event.description.present?`